src/core/reference.js

```diff
@@ -0,0 +1,344 @@
+/**
+ * @module Foundation
+ * @submodule Declaration
+ * @for p5
+ */
+
+
+/**
+
+ * Declares a block scope local variable, optionally initializing it to a value.
+ *
+ * @property let
+ * 
+ * Syntax:
+ * let var1 = value1, var2 = value2
+ * @params var1, var2, …, varN
```

Comment:
I don't think that `@params` is a tag. The non-plural `@param` can be used but only with docs generated for `@method`. Since the idea of a parameter is a bit confusing when talking about declaration I'd suggest erasing this part and trying to ensure that the assignment of value can be explained in the main text. Again, don't worry about perfecting this. I think the goal of this PR should be to get building happening correctly.